policies/trustee_roles_and_responsibilities.md

```diff
@@ -0,0 +1,127 @@
+# Farset Labs - Trustee Roles & Responsibilities
+
+Last Updated: **NOT CURRENTLY RATIFIED  POLICY OF FARSET LABS**
+
+Next Review: **NOT CURRENTLY RATIFIED  POLICY OF FARSET LABS**
+
+## What are the responsibilities of the board?
+
+* Ensuring **regular dialogue with membership** is carried out in order to act in the best interests of members and with their guidance and support.
+* Furthering the charity’s overall purpose, as set out in its [governing document](articles_of_association.md), and setting its **direction and strategy** – for example, by developing plans and strategies and monitoring progress.
+* Ensuring the work of the charity is **effective, responsible and legal** – for example, by the use of policies and procedures and systems for monitoring and evaluating the charity’s work.
+* **Safeguarding finances, resources and property** and ensuring they are used to further the charity’s purposes – for example, by insuring and documenting assets, maintaining financial systems, monitoring income and expenditure and ensuring the charity is financially sustainable or viable.
+* **Being ‘accountable’** to those with an interest or stake in or who regulate the charity – for example, by preparing annual reports and accounts and consulting with stakeholders.
+* **Being clear** about the people who carry out work on behalf of the charity (trustees, staff, volunteers), establishing and respecting boundaries between the governance role of the board and operational or day to day matters.
+* Ensuring the **board operates effectively** – for example, ensuring it receives the right reports and advice, by planning the recruitment and induction of trustees, providing trustees with support and training or carrying out reviews or appraisals of the board’s effectiveness.
+
+## Board Diversity
+
+We aim to ensure that the board of trustees represents the diversity of the community it serves. This includes maintaining a representative balance of gender, race and socio-economic backgrounds.
+
+## What are the qualities of trustees?
+
+An effective board will provide good governance and leadership by:
+
+* understanding its role and responsibilities
+* working well both as individuals and as a team
+* ensuring delivery of organisational purpose.
+* exercising appropriate control.
+* behaving with integrity and by being open and accountable.
+
+_Source: [diycommitteeguide.org](https://www.diycommitteeguide.org/)_
+
+### **Individual trustees** should be selected for
+
+* their honesty, trustworthiness, reliability and impartiality
+* their commitment to the organisation’s work
+* the skills and experience they bring.
+
+### Other key characteristics - Trustees should
+
+* understand the responsibilities they are taking on (and should be prepared to receive training if they don’t)
```

Comment:
This is an absolute **must** not a "should". Anyone joining as a trustee needs to be 100% clear on the responsibilities